firebird/mocks/__init__.py

```diff
@@ -0,0 +1,5 @@
+from pyspark import SparkConf, SparkContext
+
```

Comment:
i went back and forth on that location vs under the test dir, either is fine with me